src/pages/main/components/ProcessItem/ProcessItem.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import * as styles from './ProcessItem.css';
+
+interface ProcessItemProps {
+  icon: string;
+  title: string;
+  description: string;
+}
+
+const ProcessItem = ({ icon, title, description }: ProcessItemProps) => {
+  return (
+    <div className={styles.container}>
+      <h3 className={styles.title}>{title}</h3>
+      <div className={styles.iconWrapper}>
+        <img src={icon} alt={title} className={styles.icon} />
+      </div>
+      <p className={styles.description}>
+        {description.split('\n').map((line, index) => (
+          <React.Fragment key={index}>
+            {line}
+            {index < description.split('\n').length - 1 && <br />}
+          </React.Fragment>
+        ))}
+      </p>
```

Comment:
```suggestion
      <pre className={styles.description}>{description}</pre>
```
pre태그 사용하면 자동으로 줄바꿈처리 가능합니다!!

Reply:
우와와아앙 처음 알았어요~!!! 감사합니다유 변경했습니다